### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by hvardhan1923**
Please review week 2 assignments.

### CODE REVIEW

**Comment by ShashankSuresh** [comment]
@hvardhan1923 Can you please attach a screenshot of the output ?

File: Week-2/Assignmnet 2.2/style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+:root {
+  --blue: blue;
```
Comment (ShashankSuresh) [inline Week-2/Assignmnet 2.2/style.css]: can we use hex codes here?

File: Week-2/Assignment 2.1/Style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.row {
```
Comment (ShashankSuresh) [inline Week-2/Assignment 2.1/Style.css:1]: file name to be consistent across all files

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

File: Week-2/Assignment 2.1/Style.css (hunk 2 above)
Comment (hvardhan1923) [reply Week-2/Assignment 2.1/Style.css:1]: Sure.

File: Week-2/Assignmnet 2.2/style.css (hunk 1 above)
Comment (hvardhan1923) [reply Week-2/Assignmnet 2.2/style.css]: Yes done

**Comment by hvardhan1923** [comment]
Added screenshot of output for both exercise.

File: Week-1/Assignment 1.2/index.html [hunk 3]
```diff
@@ -1,83 +1,108 @@
...
+        <i class="las la-step-backward"></i>
+        <i class="las la-pause"></i>
+        <i class="las la-step-forward"></i>
+        <i class="las la-undo-alt"></i>
+        <i class="las la-random"></i>
+      </div>
+    </div>
+
+
+    <!-- <div class="now-playing">
```
Comment (ShashankSuresh) [inline Week-1/Assignment 1.2/index.html:90]: If commented code is not required, kindly delete them.

**Comment by ShashankSuresh** [comment]
@hvardhan1923 Please remove the screenshots from the repository and add them to the PR description. Kindly do this for any PRs which have UI activities carried out.

**Comment by ShashankSuresh** [comment]
Kindly remove week1's assignment in week2's PR

File: Week-1/Assignment 1.2/style.css [hunk 4]
```diff
@@ -1,24 +1,29 @@
-body {font-family: "Lato", sans-serif;}
+body {
+  font-family: "Lato", sans-serif;
```
Comment (ShashankSuresh) [inline Week-1/Assignment 1.2/style.css:2]: font family can also be moved to variables.

File: Week-1/Assignment 1.2/style.css [hunk 5]
```diff
@@ -1,24 +1,29 @@
...
   --black: rgb(30, 29, 29);
   --lightblue: rgb(111, 111, 217);
+
+  --font-smaller: 10px;
+  --font-small: 12px;
 }
 
 .sidebar {
-  height: 70%;
+  /* height: 70%; */
```
Comment (ShashankSuresh) [inline Week-1/Assignment 1.2/style.css:20]: Commented code to be removed

File: Week-1/Assignment 1.2/style.css [hunk 6]
```diff
@@ -138,46 +154,67 @@ tr:nth-child(even) {
...
-  margin-left: 20px;
-  color:var(--white);
+  padding-left: 10px;
+  color: var(--white);
+  font-size: var(--font-small);
 }
 
 .playlist-icon {
-  display: flex;
+  /* display: flex;
```
Comment (ShashankSuresh) [inline Week-1/Assignment 1.2/style.css:163]: Please remove if not required

File: Week-1/Assignment 1.2/style.css [hunk 7]
```diff
@@ -138,46 +154,67 @@ tr:nth-child(even) {
...
-i.las.la-step-backward {
-  font-size:24px;
-  color:var(--white);
+.playlist-icon i {
+  color: var(--white);
+  font-size: var(--font-small);
 }
 
-i.las.la-pause {
+/* i.las.la-step-backward {
```
Comment (ShashankSuresh) [inline Week-1/Assignment 1.2/style.css:177]: Kindly remove if not required

File: Week-2/Assignment 2.1/index.html [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
...apMUv5oB1LevshxHj19dtfP/H4+4OMbwdwmDeW8qfgeJIaspSuRjKSdicEf//MY1cdjKT4QQqApEbCUqYayQ23kHC4/fOO5eFEil36ta6ZF6RyUxxfUhOWmKAdStpeDj7eSeTmB24qDdqGAU7+ISR68V/2LbLkkrUkkNSAJW4fo4vBkEhakgft83Kvg4wup9zP/FtmxIkk1R6WmKCdhyspO4XE0Ffyykwmp4ok1WyJE7QhktwjwZ32hGs152bn14SFkmodcLygDZAkrEamqDq7ewsqqU5LTDUCkkT35Y6UXfz57+9KJNVniZfYAJJG8h0GSgJSBBlbUEm1WbqW91YgqaAaKZnY5GRXmaSS6gnebDUCkiSaQpqiJrElpKfgpJJqsQRIbDxJBUFbTWJLnWV6lUkuqYYBx04hgSTNFrQFKP84yb0LlaTeEihos5KKqhHl5xmkOAsqSXnr8gtCJKkgaCtMbAnJgxSFktRaAgbtvCTnk9St0sSWOtl4lalYksrRDg3aOUn5Be0cfYVnmGWa4oolKbQEqUY4kgqCtqfs/JjznX5dFyBJ3YADB+2MpKIppBpL8emDFBBJqizZ4KCdkTQyPDGBo2r/2QQM/9CBSlJjiYCqEUYSfTiSsbpZL0OwJBWWygTtzJVEpTg1Ax5uKoI3s6ANltQ+QElPtwStRros6akDrlRiSyQ5zY4tARZU0hMtlQvasaSj0yyrLXEHlfQUSyWqkbSkrdzC4/laS6yAJVUPS2UT21TSx19yvGyPXaCkypZKVSMzTv79mOO7NvkP5qjygCsftCeWtrKstArUUVVLpaoRIQP4abZMBUVVgjaH87a7XoLyjqoEbZZ+2x0vQ9ngTQbyiVcg1233uxwlLZWuRrh4bfe6LOUkKXHUoaAdU8ZR+WrkeTgqYUlR0O5SYkuAOio7hcSnU4ktARi81QTtjiW2BJClitVInrb7Wh2IpMUN2jGFihRVI112VGhpoRNbgtyRmqDd0cSWIA3eaoK213Yfn47MkpoppLZ7qALxYMOgnYBBGwLfUaV5/2friG9pwasRFl7wVuLIa7tnKmEdqalG2u6XWjCxQcDEBiHtCKsRAangraYaeUaJLSGxpKQa8druTz2oDdpt96YuVAbt55bYEkJJWI0Uof0PYGQgEDMO/q4AAAAASUVORK5CYII=" alt="Img"></div>
```
Comment (ShashankSuresh) [inline Week-2/Assignment 2.1/index.html:19]: Can we please move the img src to an actual image of format jpg or jpeg or png. Its easy to maintain the assets as opposed to using them in src.

**Comment by ShashankSuresh** [comment]
Good job on the css animations.

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Pull request merged**